src/libostree/ostree-sepolicy.c

```diff
@@ -257,6 +260,18 @@ initable_init (GInitable     *initable,
         freecon (con);
       }
 
+      {
+        g_autofree guchar *csum_raw = NULL;
+        if (!ostree_checksum_file (etc_selinux_dir, OSTREE_OBJECT_TYPE_FILE,
```

Comment:
Ahhh, I thought it was smart enough to descend in the directory and checksum the files. Hmm, OK, will do some more reading.